torch/_inductor/codecache.py

```diff
@@ -308,6 +361,59 @@ def write_atomic(path: str, content: Union[str, bytes]):
     tmp_path.rename(path)
 
 
+class FxGraphCache:
+    cache = dict()
+    clear = staticmethod(cache.clear)
+
+    @classmethod
+    def load(
+        cls, compile_fx_fn: Callable, fx_args: List[Any], fx_kwargs: Dict[str, Any]
+    ):
+        # Excluded parameters that are not stable between runs
+        fx_kwargs.pop("graph_id")
+        # Add arguments to be hashed
+        fx_args_for_hashing = copy(fx_args)
+        fx_args_for_hashing.extend(list(fx_kwargs.values()))
```

Comment:
nit: list here is not necessary
 
More substantively, taking kwargs this way is only OK assuming you always pass in the kwargs in the same order (which is the case at the call site, but may be non-obvious if there ever are other call sites)